apix-files/package.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
any reason not to house this under `packages/`? I realize there are no dependencies in either direction but still kind of seems to me like maybe we should keep all our typescript sdk-codegen packages under `packages/` (and who knows, maybe there will be a dependency some day?)

Reply:
Yes, it is not a monorepo package and doesn't need all the baggage associated with that. If it ever becomes something we customize, we'll move it into `packages/apix-files`. Currently, it's basically just a node script that needs to be isolated.

Reply:
Yeah, all those places were changed to support this so the CHANGELOG entry is appropriate.